### PR TITLE
Feat/dmb box UI

### PR DESCRIPTION
**Pull request by derianmobe**
## 📱 Working on app implementations:
  
* [ ] Upload `Screenrecord/Screenshots` from **Android** implementation.  
* [ ] Upload `Screenrecord/Screenshots` from **iOS** implementation.   

simulator_screenshot_43BC41AB-072A-485A-8261-19406A2987A3
tent.com/47223412/177200004-02c4904e-d4f9-4983-a923-d7c1d0c57433.png)

simulator_screenshot_86B45759-52BA-4B1C-A079-91CA0EF4A6F6
/177200030-64955fc3-1061-4cc6-9c35-d8b8ef5fdc50.png)

  
## 🤓 Working on native implementations:  
  
* [ ] Elaborate the change of each specific  
  
## ✚ Adding and Updating Packages? (If applicable)  
  
* [ ] Explain an elaborate the package.  
* [ ] Add to the packages list.  
  
## 💯 Quality  
  
* [ ] No new warnings or errors using `fluter analyze`.  
* [ ] All private methods start with _ prefix.  
  
**PLEASE IGNORE ANY PR THAT DOES NOT HAVE THIS DESCRIPTION UPDATED**

### CODE REVIEW

File: lib/widgets/item_cocktail.dart [hunk 1]
```diff
@@ -8,40 +10,68 @@ class CocktailItem extends StatelessWidget {
...
   final String name;
+  final bool? isTypeOfGlass;
+
+  String getImage() {
+    if (isTypeOfGlass!) {
+      return Assets.glass;
+    } else {
+      return Assets.categories;
+    }
+  }
```
Comment (keylinwu-dnamic) [inline lib/widgets/item_cocktail.dart]: We can simplify this function like:
```suggestion
  String get image =>  isTypeOfGlass ? Assets.glass : Assets.categories;
```

File: lib/widgets/item_cocktail.dart [hunk 2]
```diff
@@ -8,40 +10,68 @@ class CocktailItem extends StatelessWidget {
   const CocktailItem({
     Key? key,
     required this.name,
+    this.isTypeOfGlass = false,
   }) : super(key: key);
 
   final String name;
+  final bool? isTypeOfGlass;
```
Comment (keylinwu-dnamic) [inline lib/widgets/item_cocktail.dart]: Why does this bool needs to be optional? With a default value it doesn't need to be optional, or is there a case for it when it is actually `null`?

File: lib/widgets/item_cocktail.dart [hunk 3]
```diff
@@ -8,40 +10,68 @@ class CocktailItem extends StatelessWidget {
...
-            child: Text(name),
+          child: Row(
+            children: [
+              Image.asset(
+                getImage(),
+                width: Sizes.sizeXXL,
+              ),
+              const Padding(
+                padding:
+                    EdgeInsets.only(left: Sizes.sizeSM, right: Sizes.sizeSM),
```
Comment (keylinwu-dnamic) [inline lib/widgets/item_cocktail.dart]: Let's use spacing for this Padding cases => `Spacing .spacingSM`

File: lib/widgets/app_bar_custom.dart [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      title: Text(
+        title,
+        style: Fonts.tileTitile,
+      ),
+    );
+  }
+
+  @override
+  // TODO: implement child
+  Widget get child => throw UnimplementedError();
```
Comment (keylinwu-dnamic) [inline lib/widgets/app_bar_custom.dart]: I think this `child` override is extra here, can we double-check?

**Review by keylinwu-dnamic — CHANGES_REQUESTED** [review]

File: lib/widgets/item_cocktail.dart (hunk 2 above)
Comment (derianmobe) [reply lib/widgets/item_cocktail.dart]: There is a case for it when is null

File: lib/widgets/app_bar_custom.dart (hunk 4 above)
Comment (derianmobe) [reply lib/widgets/app_bar_custom.dart]: Need to have it because if not de custom app bar widget throws an error

**Review by keylinwu-dnamic — APPROVED** [review]

**Pull request merged**